src/serverless.js

```diff
@@ -7,17 +7,39 @@ const fs = require("fs");
 const { createOrUpdateMetaRole } = require("./utils");
 
 class LambdaCron extends Component {
+  validate(inputs) {
+    if (!inputs.schedule) {
+      throw new Error(
+        "Input 'schedule' is required. Please see README: https://git.io/JJWW0"
+      );
+    }
+    const { schedule }=inputs.schedule
+
+    let valid = false
+
+    // Check for a cron expression
+    let pat = /(@(annually|yearly|monthly|weekly|daily|hourly|reboot))|(@every (\d+(ns|us|µs|ms|s|m|h))+)|((((\d+,)+\d+|(\d+(\/|-)\d+)|\d+|\*) ?){5,7})/
```

Comment:
Hmm, these expressions are for CloudWatch events. Do they support `@` syntax? I don't think they'd allow `@reboot`, but I haven't found a canonical list. I will try to test with `@hourly` and see if it works. Same with `@every` - Although they are valid for crontab, I'm not sure they work with CloudWatch Events.

Reply:
I saw their docs. I don't think they allow. Probably better to change it with something like:
/^((((\d+,)+\d+|(\d+(\/|-|#)\d+)|\d+L?|\*(\/\d+)?|L(-\d+)?|\?|[A-Z]{3}(-[A-Z]{3})?) ?){5,7})$/